app/helpers/bulk_updates_helper.rb

```diff
@@ -172,4 +182,48 @@ def update_type_select
 
     label_tag('update-type', 'Update Type') + select_tag('update-type', options_for_select(options), prompt: 'Select an Update Type')
   end
+
+  def display_value_title(update_type, value_type)
+    data_attributes = BulkUpdatesHelper::UPDATE_TYPES.fetch(update_type.to_sym, {}).fetch(:data_attributes, {})
+
+    case value_type
+    when 'new'
+      title = data_attributes[:new_title]
+      description = data_attributes[:new_description]
+    when 'find'
+      title = data_attributes[:find_title]
+      description = data_attributes[:find_description]
+    end
+
+    "<h4>#{title}</h4><p>#{description}</p>".html_safe
+  end
+
+  def display_science_keyword(keyword)
```

Comment:
Could you please add some comments to this method, I'm confused by the `reached_value` variable specifically.